api/src/main/java/uk/gov/dhsc/htbhf/eligibility/EligibilityServiceApplication.java

```diff
@@ -22,10 +23,11 @@ public static void main(String[] args) {
     }
 
     @Bean
-    public TaskExecutor taskExecutor() {
+    public TaskExecutor taskExecutor(@Value("${threadpool.min-size}") Integer threadpoolMinSize,
```

Comment:
how about `taskExecutor.threadpool...`? Don't want it getting confused for the standard spring tomcat thread pool

Reply:
Renamed